tutorials/tagger/basic_allennlp.py

```diff
@@ -205,3 +205,45 @@ def get_metrics(self, reset: bool = False) -> Dict[str, float]:
 tag_ids = np.argmax(tag_logits, axis=-1)
 #### And then use our vocabulary to find the predicted tags.
 print([model.vocab.get_token_from_index(i, 'labels') for i in tag_ids])
+
+#### Finally, we'd like to be able to save our model and reload it later. We'll need to save two things. The first is the model weights.
+# Here's how to save the model.
+with open("/tmp/model.th", 'wb') as f:
+    torch.save(model.state_dict(), f)
+#### And the second is the vocabulary.
+vocab.save_to_files("/tmp/vocabulary")
+
+#### We only saved the model weights, so we actually have to recreate the model in code. First, let's reload the vocabulary into a new variable.
```

Comment:
Maybe something like "so we have to recreate the same model structure ourselves in code when we want to load them."?